src/query_parser/table_ref.py

```diff
@@ -0,0 +1,34 @@
+class TableInfo:
+    """
```

Comment:
I am not sure if the naming is appropriate for Eva. 

What are we referring to by a table_name here? 
Coming to the terminology, what are we referring to by schema_name and database_name. Is this applicable in the case of EVA.